### PR TITLE
Incr. Server State Validtiy to 90s

### PR DESCRIPTION
**Pull request by JeLuF**
By default, healthCheck() is run every 5s. On background tabs, this may get extended. My tests have shown pings every 60s. The ping was older than 10s, so the condition in line 490 evaluates to `false` and the client tries to access the stream before the server is ready. By increasing the validity this can be avoided - at least until the browser runs the healthcheck even less often.

See https://discord.com/channels/1014774730907209781/1041811939380178964/1041812021018120262 for the analysis.

### CODE REVIEW

**Comment by madrang** [comment]
Long tasks mitigation on desktop and mobile are so different from browser to browser.
iDevices will kill the function, Andoid slows it to a crawl, Linux will freeze it fine but on reload will redo the init again.
At least setting the timeout delay to 90 seconds will make it happen less on desktop but wont do nothing for mobile devices where the deepsleep could last many minutes before it's awaken again.

In short, you don't want to trigger long tasks mitigation by returning execution back to the browser very often.
But I did raise the SERVER_STATE_VALIDITY_DURATION inside engine.js too just in case.
I don't want to ever see that bug again.

**Comment by cmdr2** [comment]
Thanks!
